### DISCUSSION TITLE
GP - Graves

### DISCUSSION THREAD

**Discussion by ivaniraldi** [GriefPrevention/GriefPrevention#2232] [Q&A]
Hello! I hope you're doing great. I wanted to ask if there's any chance of making the plugin compatible with the Graves plugin. I'm using it with ItemsAdder, but it's throwing an error. It's not necessary for players to be able to pick up their items if they died in someone else's protection.

**Comment by QarthO** [comment]
realistically would be something that the graves plugin would need to integrate with. Have you tried asking them?

And can you link the graves plugin you're using

**Reply by ivaniraldi** [reply]
sure, i did not found anything https://www.spigotmc.org/resources/graves.74208/ (i did ask but get no answ)

**Reply by Evidentsinger14** [reply]
Graves uses ProtectionLib which should be hooking into GriefPrevention for that kind of stuff.

**Reply by ivaniraldi** [reply]
so there's no chance to have the 2 plugins working at the same time?

**Reply by QarthO** [reply]
they can definitely work together, but it would require more work on the developers of the plugins. 

Briefly looking at their code, it does look like they have some form of GriefPrevention integration. So this might just be an issue with your configuration/setup, not either of the plugins

**Comment by QarthO** [comment]
I'm not familiar with the Graves plugin. But if that's the plugin throwing errors, I would highly recommend joining their Discord and asking them about it. They would be better equipped to solving it.

**Comment by ivaniraldi** [comment]
I solved it by using the DeathChest plugin https://www.spigotmc.org/resources/death-chest.101066/
